datadog_checks_base/datadog_checks/base/checks/windows/perf_counters/counter.py

```diff
@@ -132,6 +132,9 @@ def refresh(self):
 
         for counter in self.counters:
             if counter.name not in counters:
+                self.logger.error(
```

Comment:
Should this be logged as warning instead?

Reply:
No b/c if they explicitly defined it then this would be contrary to what they need

Reply:
Makes sense, thanks!